src/client/src/ui/analytics/components/positions-chart/chartUtil.ts

```diff
@@ -137,9 +138,36 @@ export function getPositionValue(id: string, positionsData: any[]) {
   return ''
 }
 
+export function addShadow(svg: any) {
+  const defs = svg.append('defs')
+
+  const filt = defs
+    .append('filter')
+    .attr('id', 'drop-shadow')
+    .attr('height', '130%')
+
+  filt
+    .append('feGaussianBlur')
+    .attr('in', 'SourceAlpha')
+    .attr('stdDeviation', 1.5)
+    .attr('result', 'blur')
+
+  filt
+    .append('feOffset')
+    .attr('in', 'blur')
+    .attr('dx', 1)
+    .attr('dy', 1)
+    .attr('result', 'offsetBlur')
+
+  const feMerge = filt.append('feMerge')
+
+  feMerge.append('feMergeNode').attr('in', 'offsetBlur')
+  feMerge.append('feMergeNode').attr('in', 'SourceGraphic')
+}
+
 export function collide(alpha: number, nodes: any[], scale?: number) {
   const qt = d3.geom.quadtree(nodes)
-  const padding = 10
+  const padding = -3
```

Comment:
Negative padding feels a bit dirty but I don't know this API so can't off-hand suggest how else you'd do this.

Reply:
I think this is a naming issue. We relate the word padding to CSS; In this case, it seems to be an offset. This code would be better if we use long names instead of a single letter everywhere.

Reply:
Renamed this to offset + more descriptive names added